tools/nmr_annotation2d/annotationRmn2D_xml.xml

```diff
@@ -1,18 +1,17 @@
-<tool id="2DNmrAnnotation" name="2DNMRAnnotation" version="2.0.0+galaxy2" profile="20.09">
+<tool id="2DNmrAnnotation" name="2DNMRAnnotation" version="2.0.0">

```

Comment:
```suggestion
<tool id="2DNmrAnnotation" name="2DNMRAnnotation" version="2.1.0+galaxy0" profile="23.0">
```

Please keep the galaxy tool wrapper version in the tool - even though the tool is just a combination of packages. There is no dedicated NMR data processing R package used here I assume?